modules/simple_forms_api/app/services/simple_forms_api/form_remediation/s3_client.rb

```diff
@@ -117,15 +113,20 @@ def generate_presigned_url(type: upload_type)
       end
 
       def s3_upload_file_path(type)
+        extension = File.extname(archive_path)
         ext = type == :submission ? '.pdf' : '.zip'
-        build_path(:file, s3_directory_path, archive_path, ext:)
+        build_path(:file, s3_directory_path, archive_path, ext: extension ? nil : ext)
       end
 
       def presign_required?
         return true if upload_type == :submission
 
         config.presign_s3_url
       end
+
+      def manifest_required?
+        config.include_manifest && upload_type == :remediation
+      end
```

Comment:
will include_manifest always be safe to get here, even if its not defined? or its always defined?

Reply:
It will always be defined